### ISSUE TITLE
[v0.5.2-alpha] Rock walls are still missing their hitboxes

### ISSUE THREAD

**Issue by JuiceWarrior** [Teeyenoh/zephis#33]
It is possible to pass through most rock walls which are facing North/South by travelling through them from their Northern side.

Except sometimes, where you can also pass through them from the East and West. Sometimes it's only West though. 

I have no idea what's happening.

**Comment by Teeyenoh** [comment]
Yep so basically, as I was designing the N/S walls, I realised they shouldn't be in the middle of the tile and should, in fact, be to the side. As such, adding their hitboxes has been postponed until the next graphics session.

**Comment by JuiceWarrior** [comment]
This is still an issue in v0.5.3.1-alpha. No visible changes have been made with this update
